spec/workers/update_all_sources_worker_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper' 
+require 'sidekiq/testing'
+Sidekiq::Testing.fake!
+
+RSpec.describe UpdateAllSourcesWorker, type: :worker do
+    before(:each) {
+        # Criar as sources de teste.
+        @sources = [
+            Source.create(name: "Ruby", language: "en",
```

Comment:
A gente vai pegar notícias da página do Ruby? LuL

Reply:
Só pra testar :^)